lib/js/components/Headers/SectionHeader/SectionHeader.vue

```diff
@@ -0,0 +1,343 @@
+			color: $color-neutral-icon-hovered;
+		}
+
+		#{$root}__title {
+			color: $color-neutral-text-hovered;
+		}
+	}
+
+	&__iconLeft,
+	&__iconRight,
+	&__chevron {
+		color: $color-neutral-icon;
+	}
+
+	&__title {
+		color: $color-neutral-text-strong;
+		padding: $space-xxs 0;
+	}
+
+	&__info {
+		padding: $space-xxxs $space-xxs;
+	}
+
+	&.-size-l {
+		#{$root}__titleWrapper {
+			@include heading-l-default-bold;
+		}
+
+		#{$root}__iconLeft {
+			margin-right: $space-xxs;
+		}
+
+		#{$root}__iconRight,
+		#{$root}__chevron {
+			margin-left: $space-xxs;
+		}
+	}
+
+	&.-size-m {
+		#{$root}__titleWrapper {
+			@include heading-m-default-bold;
+		}
+
+		#{$root}__iconLeft {
+			margin-right: $space-xxs;
+		}
+
+		#{$root}__iconRight,
+		#{$root}__chevron {
+			margin-left: $space-xxs;
+		}
+	}
+
+	&.-size-s {
+		#{$root}__titleWrapper {
+			@include heading-s-default-bold;
+		}
+
+		#{$root}__iconLeft {
+			margin-right: $space-xxxs;
+		}
+
+		#{$root}__iconRight,
+		#{$root}__chevron {
+			margin-left: $space-xxxs;
+		}
+	}
+
+	&.-size-xs {
+		#{$root}__title {
+			padding: $space-xxxs 0;
+		}
+
+		#{$root}__titleWrapper {
+			@include heading-xs-default-bold-uppercase;
+		}
+
+		#{$root}__iconLeft {
+			margin-right: $space-xxxs;
+		}
+
+		#{$root}__iconRight,
+		#{$root}__chevron {
+			margin-left: $space-xxxs;
+		}
+	}
+
+	&__slotHorizontal {
+		display: none;
+
+		@media #{breakpoint-s()} {
+			display: block;
+		}
+	}
+
+	&__slotVertical {
+		display: block;
+		padding: 0 0 $space-xxxs;
+
+		@media #{breakpoint-s()} {
+			display: none;
+		}
+	}
+
+	&.-horizontal {
+		#{$root}__slotHorizontal {
+			display: block;
+		}
+
+		#{$root}__slotVertical {
+			display: none;
+		}
+	}
+}
+</style>
+
+<script lang="ts">
+import { SECTION_HEADER_MOBILE_LAYOUTS, SECTION_HEADER_SIZES } from './SectionHeader.consts';
+import DsIcon, { ICON_SIZES, IconItem, ICONS } from '../../Icons/Icon';
+import DsIconButton, { ICON_BUTTON_COLORS, ICON_BUTTON_SIZES } from '../../Buttons/IconButton';
+import DsDivider from '../../Divider';
+import { toRaw } from 'vue';
+
+export default {
+	name: 'SectionHeader',
+	components: {
+		DsIcon,
+		DsIconButton,
+		DsDivider,
+	},
+	props: {
+		expandable: {
+			type: Boolean,
+			default: false,
+		},
+		hideSlotWhenCollapsed: {
+			type: Boolean,
+			default: false,
+		},
+		iconLeft: {
+			type: Object as () => IconItem,
+			default: null,
+			validator(iconLeft: IconItem) {
+				return Object.values(ICONS).includes(toRaw(iconLeft));
+			},
+		},
+		iconRight: {
+			type: Object as () => IconItem,
+			default: null,
+			validator(iconRight: IconItem) {
+				return Object.values(ICONS).includes(toRaw(iconRight));
+			},
+		},
+		isExpanded: {
+			type: Boolean,
+			default: false,
+		},
+		info: {
+			type: Boolean,
+			default: false,
+		},
+		size: {
+			type: String,
+			default: SECTION_HEADER_SIZES.M,
+			validator(size) {
+				return Object.values(SECTION_HEADER_SIZES).includes(size);
+			},
+		},
+		title: {
```

Comment:
I wonder if we should validate if length of the string is greater than 0.

Reply:
Why? we have `required: true`, if someone pass an empty string then that's his problem

Reply:
I guess :)